pipeline.config

```diff
@@ -0,0 +1,53 @@
+#
+# (C) Copyright IBM Corp. 2021, 2021
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+INSIGHTS:
+  ID: "9b253889-dee0-47f4-9ea0-402306239aaa"
+CI:
+  DOCKER_IMAGE_NAME: "cohort-cql-translation-image"
+  UMBRELLA_REPO_PATH: "https://github.ibm.com/watson-health-cohorting/wh-cohorting-umbrella.git"
+  REGISTRY_NAMESPACE: "cdt-provider-cohort-rns"
+  POLICY_NAME: "CI"
+  NOLATEST: "true"
+  REPORT_PATH: "lcov.info"
+  BUILDCACHE: "false"
+  REPLACEARGS: "true"
+  VAULT_KEYS_TO_INJECT: "IBM_GITHUB_PUBLIC_USERNAME,IBM_GITHUB_PUBLIC_TOKEN"
+  # added to allow mixed case branch names (toolchains fail on mixed case branch names without this set,
+  # could remove when we switch to using main branch
+  RELEASE_NAME_NO_GIT_BRANCH: "true"
+CIVALIDATE:
+  BUILD_FILE: "pom.xml"
+  BUILD_OPTIONS: "package"
+  VAULT_KEYS_TO_INJECT: "IBM_GITHUB_PUBLIC_USERNAME,IBM_GITHUB_PUBLIC_TOKEN"
+CD:
+  POLICY_NAME: "CD"
+  APP_NAME: "cohort-cql-translation-service"
+# For a successful build to be promoted to Staging or Production, it is necessary to have an associated approved Service Now ticket.
+# The SERVICENOW_SERVICENAME value should match the "Configuration item" as defined in the servicenow change request.
+# See https://github.ibm.com/whc-toolchain/whc-commons/blob/2c9cfd2a3683dc5ba993d338f9fb0172165fb516/docs/launch/approval-gate.md
+  SERVICENOW_SERVICENAME: "wh-csp-core"
+  TILLERSERVICEACCOUNTDISABLE: "true"
+  HELM_TIMEOUT: "1800"
+  USESOURCE: "true"
+  USEHELMV3: "true"
+  VAULT_KEYS_TO_INJECT: "WH_COHORTING_APP_TOOLCHAIN_PORT,WH_COHORTING_APP_TOOLCHAIN_REPLCOUNT"
```

Comment:
I don't think these keys are used either, they came from the starter app example and were initially referenced from the helm charts I think. We should probably clean these up from the main toolchain pipeline.config file as well

Reply:
Thanks. I will kill them off.